flipt-java/src/main/java/example/Main.java

```diff
@@ -12,11 +12,11 @@ public static void main(String[] args) {
     context.put("fizz", "buzz");
 
     EvaluationRequest variantEvaluationRequest =
-        new EvaluationRequest("default", "flag1", "entity", context);
+        new EvaluationRequest("default", "flag1", "entity", context, Optional.empty());
```

Comment:
Agreed, I think (based on Ferns approach) there is precident for keeping this type how it is and adding a Builder as a sub-type for creating isntances of this final type. So this would still be a valid constructor, but there would be an additional builder to make this easier.
Does is make sense to tackle that as a general improvement in a subsequent PR?

Reply:
yeah makes sense 👍🏻

Reply:
https://github.com/flipt-io/flipt-server-sdks/pull/78